### PR TITLE
Adicionadas as funcionalidades de adicionar pessoa e remover pessoa

### PR DESCRIPTION
**Pull request by jhonatadam**
Adicionadas as funcionalidades de adicionar pessoa e remover pessoa no formulario socio economico da inscrição de auxilio moradia.

### CODE REVIEW

**Comment by jhonatadam** [comment]
legal

File: gpa-mae/src/main/webapp/resources/js/npi.js [hunk 1]
```diff
@@ -3,6 +3,42 @@ var accordionFechado;
...
+//    		}
+    		
+    	});
+    });
+    
+    $(".rmvPessoa").click(function(){
+        $(this).parent().parent().remove();
+    });
+
 	$('.fechado').slideUp();
```
Comment (npiuser) [inline gpa-mae/src/main/webapp/resources/js/npi.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2458279121/issues/source?bid=3244003&fileBranchId=3244028#l42)

Comment (npiuser) [inline gpa-mae/src/main/webapp/resources/js/npi.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2458279121/issues/source?bid=3244003&fileBranchId=3244028#l42)

File: gpa-mae/src/main/webapp/resources/js/npi.js [hunk 2]
```diff
@@ -20,6 +56,7 @@ $(document).ready(function(){
 		
 	});
 
+
 	$("#nome_cursinho").hide()
```
Comment (npiuser) [inline gpa-mae/src/main/webapp/resources/js/npi.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2458279121/issues/source?bid=3244003&fileBranchId=3244028#l60)

Comment (npiuser) [inline gpa-mae/src/main/webapp/resources/js/npi.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2458279121/issues/source?bid=3244003&fileBranchId=3244028#l60)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/AlunoController.java [hunk 3]
```diff
@@ -239,9 +238,11 @@ public String editarInscricaoIniciacaoAcademica(
...
-
+		
+		Aluno aluno = alunoService.getAlunoByCPF(auth.getName());
 		Selecao selecao = selecaoService.find(Selecao.class, idSelecao);
-
+		
+		model.addAttribute("aluno", aluno);
 		model.addAttribute("questionarioAuxilioMoradia", new QuestionarioAuxilioMoradia());
 		model.addAttribute("estado", Estado.values());
 		model.addAttribute("situacaoImovel", SituacaoImovel.values());
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/AlunoController.java]: Codacy Issue found: [The String "situacaoImovel" appears 3 times in the file.](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2458279128/issues/source?bid=3244003&fileBranchId=3244028#l248)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/AlunoController.java [hunk 4]
```diff
@@ -239,9 +238,11 @@ public String editarInscricaoIniciacaoAcademica(
...
 		model.addAttribute("action", "inscricao");
-
+		
+		Aluno aluno = alunoService.getAlunoByCPF(auth.getName());
 		Selecao selecao = selecaoService.find(Selecao.class, idSelecao);
-
+		
+		model.addAttribute("aluno", aluno);
 		model.addAttribute("questionarioAuxilioMoradia", new QuestionarioAuxilioMoradia());
 		model.addAttribute("estado", Estado.values());
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/AlunoController.java]: Codacy Issue found: [The String "estado" appears 3 times in the file.](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2458279128/issues/source?bid=3244003&fileBranchId=3244028#l247)

**Pull request closed without merging**